src/core/Form/TextInput/TextInput.test.tsx

```diff
@@ -8,10 +8,43 @@ const TestTextInput = (
   <TextInput labelText="Test input" data-testid="textinput" id="test-id" />
 );
 
+const TestTextInput2 = (
+  <TextInput
+    labelText="Test input"
+    data-testid="textinput1"
+    id="test-id1"
+    labelMode="hidden"
+    visualPlaceholder="Test TextInput"
+  />
+);
+
+const TestTextInput3 = (
+  <TextInput
+    labelText="Test input"
+    data-testid="textinput2"
+    id="test-id2"
+    visualPlaceholder="Test TextInput"
+    statusText="This is a status text"
+    status="error"
+  />
+);
+
 test('calling render with the same component on the same container does not remount', () => {
   const buttonRendered = render(TestTextInput);
   const { container } = buttonRendered;
   expect(container.firstChild).toMatchSnapshot();
 });
 
+test('calling render with the same component on the same container does not remount', () => {
```

Comment:
Would be better to have unique and more describing names for each tests.
Or at least add 1, 2, 3 if it is not easy to come with better names, as now tests look like this when ran:

```
  ✓ calling render with the same component on the same container does not remount (74ms)
  ✓ calling render with the same component on the same container does not remount (34ms)
  ✓ calling render with the same component on the same container does not remount (24ms)
```

Reply:
I agree, I think it' so quick to fix, that let's do it right away? :)

Reply:
For me the tests were automatically numbered. I added more descriptive names now.

Reply:
Now the tests names are looking nice 👍